themes/custom/apigee_kickstart/src/sass/apigee-kickstart.style.scss

```diff
@@ -98,11 +98,15 @@
 @import "apigee/apidoc";
 @import "apigee/smartdocs";
 @import "apigee/swagger";
+@import "apigee/prepaid-balance";
 @import "block/block";
 @import "form/form.contact";
 @import "form/form.search-form";
 @import "form/form.user";
-@import "form/view.exposed-form";
+@import "form/form.user";
+@import "form/form.commerce-add-to-cart";
+@import "form/form.commerce-cart";
+@import "form/form.commerce-checkout";
```

Comment:
@arshad What do you think about creating a separate CSS file/library for M10n stuff?  I'm kinda leery about making users load all this, where most will not have M10n.  We could potentially alter a commerce library to add our file too?  Curious what you think, b/c I'm also trying to figure it what is easier for people sub-theming this.